manifest: Pull LwM2M bootstrap fixes

SeppoTakalo opened the pull request:
Pull changes from Zephyr PR https://github.com/zephyrproject-rtos/zephyr/pull/61370
Related sdk-zephyr PR: https://github.com/nrfconnect/sdk-zephyr/pull/1271

NordicBuilder commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| zephyr | https://github.com/nrfconnect/sdk-zephyr/commit/c9d01d05ce83a8cec0fd60bf8e2ba27c1534d08f | https://github.com/nrfconnect/sdk-zephyr/pull/1271 | [nrfconnect/sdk-zephyr#1271/files](https://github.com/nrfconnect/sdk-zephyr/pull/1271/files) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12006
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

rlubos commented on west.yml:
```diff
@@ -59,7 +59,7 @@ manifest:
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: c9d01d05ce83a8cec0fd60bf8e2ba27c1534d08f
+      revision: pull/1271/head
```
Please update SHA: `7c57aed3643e7c10d0ce3cca96ef3c3f3387d76d`

SeppoTakalo commented:
I removed the changelog-entry tag because this bug did not exist on previous release, it only appeared last week on main, and now fixed, so it does not deserve mention.

rlubos commented:
Hmm, not sure how come the other PR got merged so fast, but the manfiest is already ahead of https://github.com/nrfconnect/sdk-zephyr/commit/7c57aed3643e7c10d0ce3cca96ef3c3f3387d76d, we can close this PR.
